app/views/teams/show.html.haml

```diff
@@ -2,42 +2,126 @@
   .listing-card-team
     %ul.collection-team
       %li.collection-team.avatar
-        .team-header
-          .summoner_name
-            %br/
-            %img.circle.responsive-img{ src: "http://ddragon.leagueoflegends.com/cdn/10.23.1/img/profileicon/#{@team.profile_image}.png", alt: "profile_img"}
-            .user_name
-              %i.material-icons person
-              %a{ href: "https://jp.op.gg/summoner/userName=#{@team.summoner_name}" } #{ @team.summoner_name }
-
-
-          .tags
-            .chip
-              = @team.rank_range
-            - @game_types.each do |g|
+        - if @team.gender == 'man'
+          .team-header-man
+            .summoner_name
+              %br/
+              %img.circle.responsive-img{ src: "http://ddragon.leagueoflegends.com/cdn/10.21.1/img/profileicon/#{@team.profile_image}.png", alt: "profile_img"}
```

Comment:
The current version of ddragon is 10.23.1 

https://developer.riotgames.com/docs/lol